bower.json

```diff
@@ -29,7 +29,9 @@
     "purescript-lists": "^0.7.0",
     "purescript-maybe": "^0.3.0",
     "purescript-strings": "~0.7.0",
-    "purescript-transformers": "^0.8.1"
+    "purescript-transformers": "^0.8.1",
+    "purescript-unicode": "git://github.com/purescript-contrib/purescript-unicode#unicode",
```

Comment:
@michaelficarra Thanks for taking a look at this PR.

I am planning to publish purescript-unicode to bower, but I am waiting for [this PR](https://github.com/purescript-contrib/purescript-unicode/pull/1) to get reviewed.

I don't think I explained it well enough in the PR comments, but this PR shouldn't be merged until purescript-unicode is released.  (Or if we decide not to use the purescript-unicode dependency, then it doesn't matter.)